wasm/wasm_wrapper.cc

```diff
@@ -40,6 +40,9 @@ void validate_options(const VW::config::options_i& options)
   if (!options.get_positional_tokens().empty()) { THROW("Positional options are not allowed") }
 }
 
+struct vw_model_basic;
```

Comment:
can we remove Line 43 - 45 and set the default MinxIn on Line183? to be consistent with cb_vw_model?

Reply:
these need to be forward declared so that vw_model can be referenced in line 55